pyccel/ast/core.py

```diff
@@ -2862,12 +2862,12 @@ class FunctionAddress(FunctionDef):
     is_kwonly: bool
         if object is an argument which can only be specified using its keyword
 
-    is_pointer: bool
-        if object is a pointer [Default value: False]
-
     is_optional: bool
         if object is an optional argument of a function [Default value: False]
 
+    memory_handling: str
+        must be \'heap\', \'stack\' or \'alias\' [Default value: 'stack']
```

Comment:
Is there a use to these backslashes? If I do:
```
In [1]: from pyccel.ast.core import FunctionAddress                                                                                                                                                        

In [2]: FunctionAddress?
```
Then I don't see the backslashes, but I still see the single quotes around the default value